app/views/books/index.html.erb

```diff
@@ -0,0 +1,15 @@
+<%= link_to("New", new_book_path, class: "btn btn-success mb-3 ml-3") %>
+<%= link_to("Users", "/users/", class: "btn btn-primary mb-3 ml-2") %>
+
+<table>
+  <thead>
+    <th class="px-3">Name</th>
+    <th class="px-3">ISBN</th>
+    <th class="px-3">Actions</th>
+  </thead>
+  <tbody>
+    <%= render(@books) %>
+  </tbody>
+</table>
+
+<%= paginate @books %>
```

Comment:
Similar to previous comments, I think we should strive for consistency here and pop some brackets in here

`<%= paginate(@books) %>`